### PR TITLE
#64 class representing schema user

### PR DESCRIPTION
**Pull request by Zejnilovic**
Closes #64

### CODE REVIEW

File: ultet/src/main/scala/za/co/absa/ultet/model/schema/SchemaCreate.scala [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * limitations under the License.
+ */
+
+package za.co.absa.ultet.model.schema
+
+import za.co.absa.ultet.model.{SQLEntry, SchemaName, TransactionGroup}
+import za.co.absa.ultet.model.TransactionGroup.TransactionGroup
+
+case class SchemaCreate(name: SchemaName) extends SQLEntry {
+  override def sqlExpression: String = s"CREATE SCHEMA $name;"
```
Comment (jakipatryk) [inline ultet/src/main/scala/za/co/absa/ultet/model/schema/SchemaCreate.scala:23]:
```suggestion
  override def sqlExpression: String = s"CREATE SCHEMA ${name.value};"
```

**Review by dk1844 — APPROVED** [review]
LGTM

**Review by jakipatryk — APPROVED** [review]

**Review by benedeki — APPROVED** [review]
Merge it, but we need to eventually add a sequence of `DBItem` that belong to the schema.
(Or create a parent class joining `DBFunction` and `DBTableDiff`)

**Comment by benedeki** [comment]
Also special handling of some schemas, like _public_

**Pull request merged**